### PR TITLE
ccmlib/scylla_node.py: use native scylla nodetool when it's available

### PR DESCRIPTION
**Pull request by tchaikov**
in b18f85d7, we use the native nodetool if JMX is not available, but when running dtest from the local build, the jmx repo is still around, so the java-based nodetool is picked. but the java-based nodetool does not support tablets, and it would be great to always test the native implementation built locally when testing using dtest without building a relocatable package.

so, in this change, we go a step further by checking if the command line of `scylla nodetool help` works, if it succeeds, we go ahead and use `scylla nodetool`, otherwise, we fall back to the java-based nodetool. we could instead use `scylla nodetool --help` instead of calling a certain command of nodetool, but seastar returns 1 if `--help` is used. a pull request was created to address it . see https://github.com/scylladb/seastar/pull/2213.

### CODE REVIEW

**Review by denesb — APPROVED** [review]

**Comment by tchaikov** [comment]
changelog

- silence the output from the stdout and stderr

**Comment by tchaikov** [comment]
this change should addrses the test failure like https://jenkins.scylladb.com/job/scylla-master/job/byo/job/dtest-byo/271/testReport/junit/nodetool_additional_test/TestNodetool/FullDtest___full_split001___test_describering/

**Comment by tchaikov** [comment]
@scylladb/qa-maintainers hello maintainers, could you help review and merge this change?

**Review by bhalevy — APPROVED** [review]
LGTM

Cc @denesb @fruch

**Review by fruch — APPROVED** [review]
LGTM

**Comment by fruch** [comment]
@tchaikov 

had to yank this out of next,

see: https://jenkins.scylladb.com/job/scylla-master/job/next/7565/testReport/junit/

**Comment by soyacz** [comment]
> @tchaikov
> 
> had to yank this out of next,
> 
> see: https://jenkins.scylladb.com/job/scylla-master/job/next/7565/testReport/junit/

I think this uncovers some issues with the new nodetool (see https://github.com/scylladb/scylladb/issues/18468)

**Comment by denesb** [comment]
Looks like there is yet another missing native nodetool command: resetlocalschema. I'm working on the implementation.

**Comment by tchaikov** [comment]
> @tchaikov
> 
> had to yank this out of next,
> 
> see: https://jenkins.scylladb.com/job/scylla-master/job/next/7565/testReport/junit/

thank you @fruch . will implement a workaround on dtest first, see https://github.com/scylladb/scylla-dtest/pull/4204. and then try to get this in after it lands.

**Pull request merged**
